app/services/doc_auth/mock/doc_auth_mock_client.rb

```diff
@@ -92,25 +86,14 @@ def get_results(instance_id:, liveness_enabled:)
         ResultResponse.new(
           self.class.last_uploaded_back_image,
           overriden_config,
-          liveness_enabled,
         )
       end
       # rubocop:enable Lint/UnusedMethodArgument
 
       private
 
-      def process_results(instance_id, liveness_checking_enabled, selfie_image)
-        results_response = get_results(
-          instance_id: instance_id,
-          liveness_enabled: liveness_checking_enabled,
-        )
-        return results_response unless results_response.success?
-
-        if liveness_checking_enabled
-          post_selfie(image: selfie_image, instance_id: instance_id).merge(results_response)
-        else
-          results_response
-        end
+      def process_results(instance_id)
```

Comment:
Should we eliminate the process_results method, simply using the get_results method?

Reply:
yeah seems worthwhile to collapse/inline, especially since it's a private method

Reply:
`process_results` eliminated